contrib/fsl_integration/sct_plugin.py

```diff
@@ -227,33 +285,496 @@ def onButtonGM(self, event):
         opts = displayCtx.getOpts(image)
         opts.cmap = 'yellow'
 
+#Computes Cross-Sectional Area for GM, WM and GM+WM (Total)
+class TabPanelCSA(SCTPanel):
+    DESCRIPTION = """This tool computes the spinal cord cross-sectional area. 
+    For more information, please refer to the article below.<br><br>
+    <b>Specific citation</b>:<br>
+    AR Martin, B De Leener, J Cohen-Adad, et al.
```

Comment:
could you please use the following convention for all citations (invert name and initials): 
Martin AR, De Leener B, Cohen-Adad J, et al.

Reply:
Please, check the references. For instance, I am not sure if I chose the right references for the dMRI.